### PR TITLE
Create namefilter.js

### PR DESCRIPTION
**Pull request by SilverTactic**

### CODE REVIEW

**Comment by panpawn** [comment]
What about a command to remove a name from the filter?

**Comment by SilverTactic** [comment]
Oh yeah. Derp

File: chat-plugins/namefilter.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	namefilter = JSON.parse(fs.readFileSync(DIR));
+} catch (err) {
+	fs.writeFileSync(DIR, '{}');
+	namefilter = JSON.parse(fs.readFileSync(DIR));
+}
+
+let notifyName = exports.notifyName = function (user) {
+	for (let i in namefilter) {
+		if (i == user.userid || user.userid.indexOf(i) > -1) {
+			Rooms('staff').add("|c|~| [NameFilter] User " + user + "'s name contains the filtered word '" + i + '"').update();
```
Comment (panpawn) [inline chat-plugins/namefilter.js]: Let's make this just a regular message. As in, make this line something like:

``Rooms('staff').add("[Name Filter] Should probably FR: <b>" + user + "</b>").update();``

**Comment by panpawn** [comment]
Any updates on the progress of this?

**Comment by panpawn** [comment]
Main is discussing a new name lock feature that would probably be more than enough moderation for names, so I'm going to close this for now.

**Pull request closed without merging**